src/pages/components/Key.js

```diff
@@ -39,25 +39,32 @@ const Row = styled.div`
   }
 `
 
-const Rows = () => (
+const Rows = ({ projectTypes }) => (
   <RowWrapper>
-    {Object.entries(projectColours).map(([key, value]) => (
-      <Row key={key} color={value}>
-        {key}
-      </Row>
-    ))}
+    {projectTypes != undefined &&
+    projectTypes != null &&
```

Comment:
Can I recommend that if `projectTypes` has no elements we just render all the colours? Other wise the key will just be empty right?

Reply:
It will default to showing all project types if `projectTypes` is empty

Reply:
Sorry misread